src/client/Ruby/ms-rest-azure/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+##2016.10.04 ms_rest version 0.6.0
```

Comment:
oh, just realized that the top comment should say ms_rest_azure intead of ms_rest for this changelog entry